<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rsa_wind</class>
 <widget class="QMainWindow" name="rsa_wind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>RSA bit</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10,10,8,12">
      <item>
       <widget class="QRadioButton" name="radioButton_2048">
        <property name="text">
         <string>2048</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_4096">
        <property name="text">
         <string>4096</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_8192">
        <property name="text">
         <string>8192</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_other">
        <property name="text">
         <string>other</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_bit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>入力</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="100,25">
      <item>
       <widget class="QLineEdit" name="lineEdit_input"/>
      </item>
      <item>
       <widget class="QLabel" name="label_input_check_bit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Public Key</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_out_pubkey">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Private Key</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_out_privkey">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>出力</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_output">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="2,10">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Padding: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>bit: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_padding">
        <property name="text">
         <string>RSA_PKCS1_OAEP_PADDING</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_status_bit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_enc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dec">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cpy">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>546</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>cpy</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
